tf_agent:
  ros__parameters:
    agent_id: 100
    dsr_input_file: ''
    source: robot
nav_agent:
  ros__parameters:
    agent_id: 101
    source: robot
docking_agent:
  ros__parameters:
    agent_id: 102
    source: robot
semantic_nav_agent:
  ros__parameters:
    agent_id: 103
    source: robot
battery_agent:
  ros__parameters:
    agent_id: 104
    source: robot
    dsr_node_name: battery
    dsr_parent_node_name: robot
    ros_topic: /battery
person_agent:
  ros__parameters:
    agent_id: 105
    source: robot
    ros_topic: object_detection/detections_3d/managed_detections
    timeout: 60
whisper_agent:
  ros__parameters:
    agent_id: 106
    source: robot
    ros_topic: /whisper/text
